README.md

```diff
@@ -1,168 +1,222 @@
 
-DSP-API and the DSP-UI library is [free software](http://www.gnu.org/philosophy/free-sw.en.html), released under the [GNU Affero General Public](http://www.gnu.org/licenses/agpl-3.0.en.html).
+DSP-API and DSP-UI-LIB are [free software](http://www.gnu.org/philosophy/free-sw.en.html), released under the [GNU Affero General Public](http://www.gnu.org/licenses/agpl-3.0.en.html).
 
-This version of DSP-UI requires [DSP-API version ^12.0.0](https://github.com/dasch-swiss/knora-api/releases/tag/v12.0.0).
+This version of DSP-UI-LIB requires [DSP-API version ^12.0.0](https://github.com/dasch-swiss/knora-api/releases/tag/v12.0.0).
 
 ## Library modules
 
-### DSP-UI/core
+### DspCoreModule
 
 The core module contains configuration files and all variables needed to connect the DSP API.
 [read more...](https://dasch-swiss.github.io/knora-ui/modules/core)
 
-* * *
+---
 
-### DSP-UI/search
+### DspViewerModule
 
-Search module allows to make simple searches or extended searches in DSP-API. In the extended search, resource class and its properties related to one specific ontology are selected to create your query.
-[read more...](https://dasch-swiss.github.io/knora-ui/modules/search)
+The viewer module contains object components to show the resource class representations from DSP-API, the GUI-elements for the property values and different kind of view frameworks.
+[read more...](https://dasch-swiss.github.io/knora-ui/modules/viewer)
 
-* * *
+---
 
-### DSP-UI/viewer
+### DspSearchModule
 
-The viewer module contains object components to show the resource class representations from DSP-API, the gui-elements for the property values and different kind of view frameworks.
-[read more...](https://dasch-swiss.github.io/knora-ui/modules/viewer)
+Search module allows to make simple searches or extended searches in DSP-API. In the extended search, resource class and its properties related to one specific ontology are selected to create your query.
+[read more...](https://dasch-swiss.github.io/knora-ui/modules/search)
 
-* * *
+---
 
-### DSP-UI/action
+### DspActionModule
 
 The action module contains special buttons (e.g. to sort a list), pipes and directives.
 [read more...](https://dasch-swiss.github.io/knora-ui/modules/action)
 
-* * *
+---
 
 ## Developers note
 
 ### Prerequisites
 
-We develop the DSP-UI modules with Angular 8, especially with Angular-cli, which requires the following tools:
+We develop the DSP-UI-LIB modules with Angular 9, especially with Angular-cli, which requires the following tools:
+
+#### Node.js
 
-#### Node.js / NPM
+Angular requires a [current, active LTS, or maintenance LTS](https://nodejs.org/about/releases/) version of Node.js. We recommend to install [Node version 12.x](https://nodejs.org/download/release/latest-v12.x/).
```

Comment:
I think the node version should go in a separate file, see my comment above

Reply:
Yes, but someone has to write it...

Reply:
https://dasch.myjetbrains.com/youtrack/issue/DSP-185